worker/include/RTC/RTCP/FeedbackPsTst.hpp

```diff
@@ -36,7 +42,11 @@ namespace RTC
 				uint8_t reserved2 : 3;
 				uint8_t index : 5;
 #endif
-			};
+			} MEDIASOUP_PACKED;
```

Comment:
This is completely unknown for me, are we using the same somewhere else?

Reply:
The goal here is to have packed struct, such that compiler doesn't add extra alignment. However, as of C++11 standard there is no canonical way of doing it and different compilers have their own syntax.

MSVC:
```cpp
// Align to 1 byte
#pragma pack(push, 1)
struct X {};
// Revert back to previous behavior
#pragma pack(pop) 
```

GCC:
```cpp
struct X {} __attribute__((packed));
```

I took this pattern from usrsctp codebase that has a lot of packed data structures.